Test.Utility/TestEnvironmentProvider.cs

```diff
@@ -53,21 +48,15 @@ public void Dispose()
                 File.SetAttributes(destinationPath, FileAttributes.Normal);
             }
 
-            Directory.Delete(RootDirectory, true);
+            _tempDirectory.Dispose();
         }
-
-        private static string GenerateParent()
+        

```

Comment:
Remove whitespace on this line

Reply:
https://github.com/messerli-informatik-ag/test-utility/pull/7/commits/f68766a81c35103374e1365b9d78dfae67a5344f